classes.js

```diff
@@ -0,0 +1,54 @@
+class Human {
+  constructor(name, age){
+   this.name = name;
+   this.age = age;
+
+  }
+  ageOneYear () {
+  return this.age++;
+  }
+
+  eating () {
+    return "mmm, mmm, mmm, I'm love'n it";
+  }
```

Comment:
````
Add the function eating, that logs "mmm, mmm, mmm, I'm love'n it".
```
You will want your eating function to LOG the line to the console, rather than just returning it:
`console.log("mmm, mmm, mmm, I'm love'n it")`
This way, when you call the function, it will automatically log the line for you.